tools/pd-simulator/simulator/node.go

```diff
@@ -47,6 +48,9 @@ type Node struct {
 	cancel                   context.CancelFunc
 	raftEngine               *RaftEngine
 	ioRate                   int64
+	available                *uint64
```

Comment:
Why we need it?

Reply:
This is a very inelastic way to pass parameters to the store .

Reply:
It has been packaged